guides/common/attributes.adoc

```diff
@@ -169,9 +171,11 @@ ifeval::["{build}" == "satellite"]
 :ansible-namespace: `redhat.satellite`
 :ansible-namespace-example: `redhat.satellite._module_name_`
 :awx: Ansible Tower
+:index: 
```

Comment:
This is confusing, I suggest that the `index` variable holds either `index.html` or `index-foreman.html`. URLs will also look a bit cleaner.

Reply:
Good notice, will do